src/index.tsx

```diff
@@ -17,6 +18,8 @@ import AppProviders from './context';
 //     console.log('  Deet > ', { inter, phase });
 // };
 
+initLogRocket();
```

Comment:
Not sure the best place to call this but decided as early as possible was probably best.